Source/CombatExtended/CombatExtended/StatWorkers/StatWorker_ArmorPartial.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using RimWorld;
+using UnityEngine;
+using Verse;
+
+namespace CombatExtended
+{
+    public class StatWorker_ArmorPartial : StatWorker
+    {
+        public override string GetExplanationFinalizePart(StatRequest req, ToStringNumberSense numberSense, float finalVal)
+        {
+            if (req.Thing.def.HasModExtension<PartialArmorExt>())
+            {
+                var result = "General value: " + finalVal.ToString() + " \n \n Special values for bodyparts ".Colorize(Color.red);

```

Comment:
Would be better to use localized strings, to make translation easier, rather than hard coding English.